### ISSUE TITLE
"Souveränität" vs "Hoheit"

### ISSUE THREAD

**Issue by od-ms** [od-ms/datennutzungsklauseln-muster#2]
"datenhoheit" scheint fuer eine stadt der bessere begriff. klingt zwar altmodisch traditionell, ist aber nicht mehrdeutig wie "souveraenitaet" die es staatlich, persoenlich u. f. juristische personen gibt. bei nutzerinnen und system waere "daten-autonomie" geeignet.

https://twitter.com/pitsch/status/1387806613483003906?s=19

**Comment by stkdiretto** [comment]
Sehe die Verwendung des Souveraenitaets-Begriffs auch schwierig. Was ist der eigentliche Zweck, den man erreichen will? Ich verstehe es so, dass die Stadt ueber die von ihr bzw. in ihrem Auftrag erhobenen Datenpunkte frei verfuegen und auch die Nachnutzung beliebig freigeben kann. Liege ich damit richtig?

**Comment by od-ms** [comment]
@stkdiretto Ja, genau, das ist der Zweck. Wir wollen sicherstellen, dass wir nicht in den Nutzungsmöglichkeiten der städtischen Daten eingeschränkt werden.

**Comment by stkdiretto** [comment]
Dann waer ich aber mittlerweile echt fuer einen Begriff wie „Nutzungsfreizuegigkeit“ oder „Sicherung vor Privatisierung“ oder sonstwas. Ich hab mich in der Zwischenzeit in Metaliteratur eingelesen, wie krass der Begriff verwendet wird, und das wird immer wilder :D

**Comment by od-ms** [comment]
Gedanken: 
 - Eigentlich unser Ziel: "Wir wollen als Kommune digital souverän sein."
 - Aber: "Datensouveränität" wird in der Tat in aktuellen Medien schwammig verwendet, ist erklärungsbedürtig.
 - "Sicherung vor Privatisierung" ist für die Überschrift zu provokativ.
 - "Nutzungsfreizuegigkeit" ist auch erklärungsbedürftig
 - "Datenhoheit" ist vielleicht für die Zielgruppe am griffigsten
